leaflet-map.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+	'use strict';
+	kintone.events.on('app.record.detail.show', function(event) {
+		const record = event.record;
+		const address = record.Address.value;
+		const headerSpace = kintone.app.record.getHeaderMenuSpaceElement();
+		const kintoneAddress = [37.789808, -122.401767];
+		const name = record.Place.value;
+
+		const mapSpace = document.createElement('div');
+		mapSpace.style.height = '300px';
+		mapSpace.setAttribute('id', 'kintone');
+		headerSpace.appendChild(mapSpace);
+
+		const kintoneMap = L.map('kintone').setView(kintoneAddress, 16);
```

Comment:
Define a variable that stores the value of the id and refer to it instead of specifying `'kintone'` as a string.